### PR TITLE
CI configuration with travis

### PR DESCRIPTION
**Pull request by allegroCoder**

### CODE REVIEW

**Comment by allegroCoder** [comment]
@snowleopard CI set with travis

**Comment by allegroCoder** [comment]
Can I also remove the `fantasi-backend` repository from tuura?

File: README.md [hunk 1]
```diff
@@ -1,3 +1,4 @@
+[![Build Status](https://travis-ci.org/tuura/fantasi.svg?branch=CI-travis)](https://travis-ci.org/tuura/fantasi)
```
Comment (snowleopard) [inline README.md:1]: Do I have to set up CI at Travis or can you do it yourself?

**Comment by snowleopard** [comment]
@allegroCoder Thanks!

Regarding the `fantasi-backend`: @thisiswhereitype can you delete it? It looks like Alessandro has taken care of migrating all the code to this repo.

File: README.md (hunk 1 above)
Comment (allegroCoder) [reply README.md:1]: I did it myself, I didn't need any permissions :)

**Comment by snowleopard** [comment]
Merged!

**Pull request merged**